### PR TITLE
flex ot grid

### PR DESCRIPTION
**Pull request by Evgenb000**
- [DEMO LINK](https://Evgenb000.github.io/layout_catalog_grid/)

### CODE REVIEW

**Review by BondarenkoDmitriy — APPROVED** [review]
no comments, good job!

**Review by VinnieJ-2k20 — CHANGES_REQUESTED** [review]
Layout is broken on ~960 px resolution and below
image

File: src/styles/_catalog.scss [hunk 1]
```diff
@@ -1,22 +1,28 @@
...
-  flex-wrap: wrap;
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(1, 200px);
+  gap: 48px;
 
-  max-width: 944px;
-  margin: 0 auto;
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, 200px);
```
Comment (G-YaSha) [inline src/styles/_catalog.scss:8]: You used `200px` value in several places, so it makes sense to create a separate variable for it

**Review by G-YaSha — APPROVED** [review]
Good job 👍 Make sure to fix comment👇

**Pull request closed without merging**